app/scripts/modules/core/application/config/applicationAttributes.directive.html

```diff
@@ -16,11 +12,31 @@
             <dd ng-if="vm.application.attributes.repoProjectKey">{{vm.application.attributes.repoProjectKey}}</dd>
             <dt ng-if="vm.application.attributes.repoSlug">Source Repo</dt>
             <dd ng-if="vm.application.attributes.repoSlug">{{vm.application.attributes.repoSlug}}</dd>
+            <dt>Description</dt>
+            <dd>{{vm.application.attributes.description}}</dd>
+            <dt>Account(s)</dt>
+            <dd>{{vm.application.attributes.accounts}}</dd>
             <dt ng-if="vm.application.attributes.cloudProviders">Cloud Provider(s)</dt>
             <dd ng-if="vm.application.attributes.cloudProviders">{{vm.application.attributes.cloudProviders}}</dd>
+            <dd ng-if="vm.application.attributes.cloudProviders">{{vm.application.attributes.cloudProviders}}</dd>
+            <dt
+              ng-if="vm.application.attributes.platformHealthOnly || vm.application.attributes.platformHealthOnlyShowOverride">
+              Instance health
+            </dt>
+            <dd ng-if="vm.application.attributes.platformHealthOnly">
+              This application considers only cloud provider health when executing tasks
+            </dd>
+            <dd ng-if="vm.application.attributes.platformHealthOnlyShowOverride">
+              This application will show a health override option for each operation
+            </dd>
```

Comment:
Don't you want to use the message you set on the controller i.e. `{{vm.healthMessage}}`, like you're doing in `app/scripts/modules/netflix/application/applicationAttributes.directive.html` below?